### DISCUSSION TITLE
nav bar in base template and li element

### DISCUSSION THREAD

**Discussion by nabajour** [getnikola/nikola#3640] [Q&A] [answered]
Hi, 

I just started using Nikola and am working on writing a template for my website. 

While trying to understand the navigation menu in the `base`template, I was wondering what [this](https://github.com/getnikola/nikola/blob/1f487e689eb22fee565b629ddbad5f8565a3def1/nikola/data/themes/base/templates/base_header.tmpl#L44) `<li>` element in the `html_navigation_links_entries` function in `base_headers.tmpl` is for. 

It looks to me that it's either superfluous or there is a `</li>` lacking. 

Or am I mising something?

Thanks,

nab

**Comment by ralsina** [comment, marked as answer]
<li> elements don't need to be closed. This is perfectly valid:

<ul>
<li> An item
<li> Another item
<li> And a third one
</ul>

The spec says:

"An li element's end tag may be omitted if the li element is immediately
followed by another li element or if there is no more content in the parent
element."

In that particular case, it's a list where each element is an item which
(in the case of submenus) also contains a list.

**Reply by nabajour** [reply]
Ah, thanks! Didn't know that and it bugged my sense of symmetry :-D
